doc-content/jbpm-docs/src/main/asciidoc/BPMN2/appe-service-tasks.adoc

```diff
@@ -295,10 +295,10 @@ Content::
 The data you want to send. This attribute is mandatory for POST and PUT requests. This is an optional parameter. If you want to use it, map it as a data input variable in the *Data I/O* dialogue of the task.
 
 ConnectTimeout::
-The connection timeout. The default value is 60 seconds.
+The connection timeout. The default value is 60000 milliseconds/60 seconds. The input value must be provided in milliseconds.
```

Comment:
```suggestion
The connection timeout. The default value is 60000 milliseconds or 60 seconds. You must provide the input value in milliseconds. 
```
Avoid forward or backward slashes in the content. For more about Slashes, check our new IBMSG page no 106.